clients/js/src/ChromaClient.ts

```diff
@@ -290,7 +290,7 @@ export class ChromaClient {
   /**
    * Lists all collections.
    *
-   * @returns {Promise<CollectionType[]>} A promise that resolves to a list of collection names.
```

Comment:
i dont think this is the case - can we verify if we have not already? I think it does return a list of `CollectionType`. This is unrelated from export `Collection` too?

Reply:
You are correct! This was meant to be consistent with Python client, but the Python client has this bug - it returns the actual collections. Changing this to only return collection names until we have EF persistence and formal collection configuration.